### PR TITLE
Alternative performance increase

### PR DESCRIPTION
**Pull request by WyriHaximus**
As suggested by @bwoebi at https://github.com/igorw/evenement/pull/39#issuecomment-315599713

### CODE REVIEW

File: src/Evenement/EventEmitterTrait.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public function once($event, callable $listener)
         $onceListener = function () use (&$onceListener, $event, $listener) {
             $this->removeListener($event, $onceListener);
 
-            call_user_func_array($listener, func_get_args());
+            \call_user_func_array($listener, \func_get_args());
```
Comment (kelunik) [inline src/Evenement/EventEmitterTrait.php]: You could add the `...$args` here, too, but then you have to bump the minimum version.

**Comment by clue** [comment]
Nice! I see there's some discussion going on about this on IRC, do you care to share some details here for those not following through? :+1: 

Perhaps we can compare some benchmarking results for both alternatives and decide if we want either or both and which version they could target. Also, a deeper understanding of why certain changes have a significant effect on performance would help here, also to avoid any regressions in the future.

The suggested change looks like a micro optimization to me and makes me wonder if this has such a noticeable effect across all PHP versions?

File: src/Evenement/EventEmitterTrait.php (hunk 1 above)
Comment (bwoebi) [inline src/Evenement/EventEmitterTrait.php]: He could, but it wouldn't really bring a perf increase.

Comment (WyriHaximus) [inline src/Evenement/EventEmitterTrait.php]: I tried that and it doesn't at all.

**Comment by WyriHaximus** [comment]
@clue The benchmarks are ran on Travis. Just select a job and scroll to the end:
master: https://travis-ci.org/igorw/evenement/builds/254238504
[without call_user_func](https://github.com/igorw/evenement/pull/39): https://travis-ci.org/igorw/evenement/builds/254122122
[\call_user_func](https://github.com/igorw/evenement/pull/42): https://travis-ci.org/igorw/evenement/builds/254240283

**Comment by WyriHaximus** [comment]
Merging this one as the breaking changes will be introduced through #39

**Pull request merged**